<launch>
    <node name="wifi_localization" pkg="wifi_localization" type="wifi_localization" output="screen">
        <param name="path_to_csv" type="string" value="/informatik2/students/home/2scholz/BachelorArbeit/catkin_ws/wifi_data/important/08_17_hallway_full" />
        <param name="n_particles" type="int" value="500" />
    </node>
</launch>